test/common/flags.test.ts

```diff
@@ -134,4 +134,24 @@ describe('requiredDoceOrgFlag', () => {
       expect(err.message).to.include(`No authorization information found for ${invalidAlias}`);
     }
   });
+
+  it('fails when no alias is provided and the --target-devops-center config var is set but invalid', async () => {
+    const invalidTargetDevopsCenter = 'invalid-target-devops-center';
+    sandbox.stub(ConfigAggregator.prototype, 'getInfo').returns({
+      value: 'invalid-target-devops-center',
```

Comment:
```suggestion
      value: invalidTargetDevopsCenter,
```

Reply:
In this case that it's a dummy hardcoded value for testing and variable naming conventions don't apply I like to use kebab casing for readability.

Reply:
I understand. I thought we could also use the constant declared above here.

Reply:
Can't use the same constant because here I want to test the case where the given value doesn't match a valid authenticated org.

Reply:
Ok